config/diff.go

```diff
@@ -16,12 +16,15 @@ import (
 // where left is usually old and right is new. So the diff is the
 // changes from left to right.
 type Diff struct {
-	Left, Right         *Config
-	Added               *Config
-	Modified            *ModifiedConfigDiff
-	Removed             *Config
-	ResourcesEqual      bool
-	NetworkEqual        bool
+	Left, Right    *Config
+	Added          *Config
+	Modified       *ModifiedConfigDiff
+	Removed        *Config
+	ResourcesEqual bool
+	NetworkEqual   bool
+	// LogFieldEqual is a field to determine if an appropriate warning about Log
+	// changes needs to be displayed.
+	LogFieldEqual       bool
```

Comment:
feels like instead of adding a new field, LogEqual should be changed to reflect only log changing, and maybe the current usage of log equal should be offloaded into a function that is ResourcesEqual OR LogEqual